recipes/orator/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "orator" %}
+{% set version = "0.9.9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 6fe7830c40f20e77929b80b741a3b9f2145634b5f411176ecad0e761fef26f55
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+    - poetry
+  run:
+    - python
+    - psycopg2
+    - mysqlclient
+    - backpack
+    - pendulum
```

Comment:
Can you please try `pendulum <2` instead? This might be related, https://stackoverflow.com/questions/57616144/cant-create-exe-with-pycharm-as-it-complains-pendulum-locales.

Reply:
Works well thanks, I brought the poetry versions across and it seems to work locally now.